requirements.txt

```diff
@@ -9,3 +9,7 @@ scipy
 requests>=2.25,<3
 numpy>=1.19
 protobuf>=3.15.0,<4
+transformers
```

Comment:
please remove all these additional from requirements.txt! We cannot make this a global dependency. Use the extra dependency in setup.py for that